QMLComponents/components/JASP/Controls/VariablesForm.qml

```diff
@@ -33,7 +33,15 @@ VariablesFormBase
 	preferredHeight				: implicitHeight
 	preferredWidth				: implicitWidth
 	Layout.preferredHeight		: preferredHeight // Cannot set Layout attached property in c++
-	focusOnTab					: false
+
+	onActiveFocusChanged		:
+	{
+		if (activeFocus) {
+			availableVariablesList.forceActiveFocus();
+			//the first button knows the way out...
+			availableVariablesList.KeyNavigation.backtab = assignButtonRepeater.itemAt(0).nextItemInFocusChain(false);
+		}
```

Comment:
I'm not sure to understand this code... How can the first assign button know which control should get the focus after a backtab on the Available VariablesList?

Reply:
made some changes to clarify